src/components/RepoItem/RepoItem.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { useDispatch } from 'react-redux'
+import { removeRepository } from '../../actions'
+import { useHistory } from 'react-router-dom'
+import './RepoItem.scss'
+
+const RepoItem = ({id, name, starsCount}) => {
+  let history = useHistory()
+  const dispatch = useDispatch()
+
+  const handleRemove = (e) => {
+    e.preventDefault()
+    dispatch(removeRepository(id))
+  }
+ 
+  return(
+    <tr
+      className='repo-item'
+      onClick={() => history.push(`/repositories/${id}`)}>
```

Comment:
You can wrap component with `Link` from `react-router-dom` which handles all routing

Reply:
Couldn't do that, because the component is a row inside table